libnavui-dropin/src/main/res/layout/mapbox_info_panel_header_layout.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:background="@drawable/mapbox_bg_info_panel"
+    tools:parentTag="android.widget.FrameLayout">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/mapbox_infoPanel_peekHeight">
+
+        <View
+            style="@style/MapboxStyle.DropIn.DragHandle"
+            android:id="@+id/handle"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent" />
+
+        <com.mapbox.navigation.ui.tripprogress.view.MapboxTripProgressView
+            android:id="@+id/tripProgressView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="20dp"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent" />
```

Comment:
This can be addressed as tailwork

We want to allow the `MapboxTripProgressView` to be injected by itself. Right now we have the entire info_panel_header injected, but we want to be able to replace the `MapboxTripProgressView`.

So convert this into a FrameLayout and expose a `TripProgressBinder`

Reply:
Good idea. Would you like me to do this as a part of this PR?

Reply:
Yeah can you? That would be 💯